paseos/actors/actor_builder.py

```diff
@@ -152,6 +152,12 @@ def set_power_devices(
             actor, SpacecraftActor
         ), "Power devices are only supported for SpacecraftActors"
 
+        # Check if the actor already had a power device
+        if actor.has_power_model:
+            logger.warning(
+                "The actor already had a power device. Currently only one device is supported. Overriding old device."
+            )
+
```

Comment:
Please, add a statement about to the README where you explain that it is possible to add only power device at the time. Trying to add more than one will lead to override the first.